backend/app/helpers/spree/admin/navigation_helper.rb

```diff
@@ -151,10 +151,12 @@ def link_to_with_icon(icon_name, text, url, options = {})
         link_to(text.html_safe, url, options)
       end
 
-      def icon(icon_name)
+      def spree_icon(icon_name)
         icon_name ? content_tag(:i, '', class: icon_name) : ''
       end
 
+      alias :icon :spree_icon
```

Comment:
@bbonislawski we need deprecation warnings for usage of the old methods so other developers will know about this change and that they need to reflect it in their custom frontends if they do use it